### DISCUSSION TITLE
Feature Request: Role Based Chart Access

### DISCUSSION THREAD

**Discussion by jenarosm** [apache/superset#24812] [Ideas]
Adding the possibility to either allow certain roles to access and modify charts or make the charts modifiable by parent dashboard owners. 

Currently in order to modify a chart as an admin you need to grant youself access inside the chart properties.
![image](https://github.com/apache/superset/assets/51678677/d18aa8a5-5f62-4c65-8c99-3da11a214a06)

It would certainly be useful to be able to apply the same Role access the same way you can within a dashboard.
![image](https://github.com/apache/superset/assets/51678677/f2c2c41a-1667-4897-bcf0-c369b75a5331)

If there is an easier way for managing access control of multiple charts in bulk, i am currently unaware of it.

**Comment by yinebebt** [comment]
Hi everyone,
I’m currently facing a similar issue where I need to update ownership or access permissions for multiple entities (charts, dashboards, datasets) in bulk. Right now, I have to go through each entity individually, which is time-consuming.

Is there a way to add owner access to multiple users (or even a single user) for multiple entities at once? For example, updating ownership for a batch of charts in one go.

Thanks!
